### DISCUSSION TITLE
What's the difference between `istio` and `istio-waypont` in GatewayClass

### DISCUSSION THREAD

**Discussion by Rory-Z** [istio/istio#54495] [Q&A] [answered]
Hi team, I'm a beginner for istio.

I want deploy to Gateway for ServiceEntry to controller engress rules, and I also want this Gateway can controller ingress rules use VirtualService and other Kubernetes Gateway API.

And I see in your documents, some Gateway use `istio-waypont` GatewayClass, and some Gateway use `istio` Gateway, it's makes me confuse.

So what's the difference between `istio` and `istio-waypont` in GatewayClass ? In other gateway controller, like traefik，they just have once GatewayClass, but istio have three GatewayClass, `istio` and `istio-waypoint` and `istio-remote`

**Comment by howardjohn** [comment, marked as answer]
`istio` is for ingress (https://istio.io/latest/docs/tasks/traffic-management/ingress/gateway-api/). So accepting external traffic typically.

`istio-waypoint` is to deploy waypoint, so for easy-west traffic.

**Reply by Rory-Z** [reply]
Can I use once Gateway work for accepting external traffic and east-west traffic in the same time ? If yes, which GatewayClass should I use ? or must use two different Gateway for work ?

**Reply by Rory-Z** [reply]
Hi @howardjohn Could you please take a look

**Reply by howardjohn** [reply]
Currently you need two separate gateways
